opm/autodiff/BlackoilDetails.hpp

```diff
@@ -100,60 +100,18 @@ namespace detail {
         /// \param end             end iterator for the given vector
         /// \param num_components  number of components (i.e. phases) in the vector
         /// \param pinfo           In a parallel this holds the information about the data distribution.
-        template <class Iterator>
+        template <class Iterator, class Communicator>
         inline
-        double euclidianNormSquared( Iterator it, const Iterator end, int num_components, const boost::any& pinfo = boost::any() )
+        double euclidianNormSquared(Iterator it,
+                                    const Iterator end,
+                                    int num_components,
+                                    const Communicator& comm = Dune::CollectiveCommunication<Dune::No_Comm>() )
```

Comment:
Would prefer to avoid the default argument here, the helper provides the same communicator for the serial case anyway in a simple way.